src/poprox_recommender/recommenders/configurations/mmr.py

```diff
@@ -0,0 +1,26 @@
+# pyright: basic
+
+from lenskit.pipeline import PipelineBuilder
+
+from poprox_recommender.components.diversifiers import (
+    # LocalityCalibrator,
+    MMRDiversifier,
+)
+from poprox_recommender.components.embedders import NRMSArticleEmbedder, NRMSUserEmbedder
+from poprox_recommender.paths import model_file_path
+
+from .. import common
```

Comment:
I get that there are repeated chunks of code across the pipelines, but this `common` thing seems a little weird to me. It obscures the structure of the pipeline by making the reader bounce between files to figure out what's connected to what. Reversing how `builder` is used from the normal `builder.do_whatever()` style is kinda confusing too.

I'd probably just go ahead and duplicate the code across files for now, figuring that's an acceptable intermediate step between where we were and loading a directory of serialized pipeline definition files

Reply:
Yeah, if we do want to abstract a `PoproxPipelineBuilder` class that wraps `PipelineBuilder` is probably a better plan. I'll duplicate the code for now.